internal/config/config_type.go

```diff
@@ -226,6 +226,11 @@ func NewBlankRoot() *yaml.Node {
 										Kind:        yaml.ScalarNode,
 										Value:       "api_protocol",
 									},
+									{
+										HeadComment: "Configure host for api endpoint, defaults to the host itself",
+										Kind:        yaml.ScalarNode,
+										Value:       "api_host",
+									},
```

Comment:
With this, you've only set the key without the value so it reads as `api_protocol: api_host` causing most tests to fail.
Here is a suggestion:

```suggestion
									{
										HeadComment: "Configure host for api endpoint, defaults to the host itself",
										Kind:        yaml.ScalarNode,
										Value:       "api_host",
									},
									{
										Kind:  yaml.ScalarNode,
										Value: "gitlab.com",
									},
```
It should also be on above the api_protocol (i.e before line 224)